### PR TITLE
noSig for informa:informa:0.6.0

### PR DESCRIPTION
**Pull request by aoindustries**
This is the only artifact for groupId "informa".

### CODE REVIEW

File: resources/pgp-keys-map.list [hunk 1]
```diff
@@ -216,6 +216,8 @@ geronimo-spec                   = noSig
 
 info.picocli                    = 0x8756C4F765C9AC3CB6B85D62379CE192D401AB61
 
+informa:informa:0.6.0           = noSig
```
Comment (slawekjaranowski) [inline resources/pgp-keys-map.list:219]: There is only this version
https://repo1.maven.org/maven2/informa/informa/
so specific versions range here is not need now

**Review by slawekjaranowski — CHANGES_REQUESTED** [review]

File: resources/pgp-keys-map.list (hunk 1 above)
Comment (aoindustries) [reply resources/pgp-keys-map.list]: I have rebased and removed the version range.

**Review by slawekjaranowski — DISMISSED** [review]

**Comment by slawekjaranowski** [comment]
please resolve conflicts

**Review by slawekjaranowski — DISMISSED** [review]

**Comment by aoindustries** [comment]
Rebased, no other changes.  Sorry if this gives you more work to approve again.

**Review by slawekjaranowski — APPROVED** [review]

**Comment by slawekjaranowski** [comment]
@aoindustries  - If you wish I can release next version. Let me know.

Once again thank you very much for your collaborations.

**Comment by aoindustries** [comment]
#response_container_BBPPID{font-family: initial; font-size:initial; color: initial;}  We have a few more to go.                                                                                                                                             Dan ArmstrongAO Industries, ***@***.***Work: (251) 607-9556Cell: (205) 454-2556                                                                                                                                                                           From: ***@***.***Sent: July 10, 2021 10:08 AMTo: ***@***.***Reply-to: ***@***.***Cc: ***@***.***; ***@***.***Subject: Re: [s4u/pgp-keys-map] noSig for informa:informa:0.6.0 (#175)  
@aoindustries  - If you wish I can release next version. Let me know.
Once again thank you very much for your collaborations.

**Pull request merged**